vector/src/main/java/im/vector/app/features/home/room/list/home/NewChatBottomSheet.kt

```diff
@@ -53,6 +56,12 @@ class NewChatBottomSheet @Inject constructor() : BottomSheetDialogFragment() {
         }
     }
 
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        return super.onCreateDialog(savedInstanceState).apply {
+            (this as BottomSheetDialog).setPeekHeightAsScreenPercentage(0.5f)
+        }
+    }
```

Comment:
All those new BottomSheet should extend `VectorBaseBottomSheetDialogFragment`, for analytics support and common behavior. Those specific treatment could then be moved to the parent, to avoid code duplication.

Reply:
Happy to make the changes 👍

Reply:
thanks!